servers/rendering/rendering_server_canvas.cpp

```diff
@@ -545,6 +545,14 @@ void RenderingServerCanvas::canvas_item_add_polyline(RID p_item, const Vector<Po
 	Item *canvas_item = canvas_item_owner.getornull(p_item);
 	ERR_FAIL_COND(!canvas_item);
 
+	Color color;
+	Color color2;
+
+	if (p_colors.empty()) {
+		color = Color(1, 1, 1, 1);
+		color2 = Color(1, 1, 1, 0);
+	}
```

Comment:
I guess you could do directly
```suggestion
	Color color = Color(1, 1, 1, 1);
	Color color2 = Color(1, 1, 1, 0);
```

Reply:
Fixed